src/test/java/com/shapesecurity/bandolier/es2017/Test262.java

```diff
@@ -404,7 +409,13 @@ public void testTest262() throws Exception {
 		Path root = Paths.get(testsDir);
 		Files.walk(root).forEach(path -> {
 			try {
-				runTest(root, path);
+				int hash = runTest(root, path);
+				for (int i = 0; i < 2; ++i) {
+					int localHash = runTest(root, path);
```

Comment:
I would prefer we not actually evaluate the test on subsequent runs. Ideally we'd split out the "bundle" portion from the "execute" portion, and only perform the "bundle" portion here.

Reply:
Updated